readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -418,8 +418,8 @@ def test_get_invalid_raw_log(self):
 class APITests(TestCase):
     fixtures = ['eric.json', 'test_data.json']
 
-    def test_make_project(self):
-        """Test that a superuser can use the API."""
+    def test_cant_make_project(self):
+        """Test that a user can't use the API to create projects."""
```

Comment:
You inverted the logic of the test here.

Don't we still need the test for testing that it's possible to use the API as super user to create a project?

Reply:
We decided to remove the creation of a project using the api, it was never documented anyway.

Reply:
On this change specifically, there's more backstory in https://github.com/rtfd/readthedocs.org/pull/4325.